.github/settings.yml

```diff
@@ -0,0 +1,36 @@
+repository:
```

Comment:
This is mostly taken from our template repo file with the changes of no labels and using `teams` instead of collaborators (which properly configures the repo settings). I found out that if you set labels then it wipes _all_ labels from the repo and only allows those that are set in the probot settings file (not ideal).